content/posts/networkx/node-ordering-Ti-updating/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Updates on VF2++"
+date: 2022-07-06
+draft: false
+description: "
+Summary of the progress on VF2++
+"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["Konstantinos Petridis"]
+
+---
+
+This post includes all the major updates since the last post about VF2++. Each section is dedicated to a different
```

Comment:
Add a link to the previous post. You can have relative links `[here]({{< relref "../GSoC-2022" >}}`.

I would also update the first post with a table of content at the beginning. We can have the same table in all posts. Something similar is found in other blogs when there are follow up.